README.md

```diff
@@ -20,9 +20,11 @@ export MAVEN_OPTS="-Xmx4g -XX:MaxPermSize=1024M -XX:ReservedCodeCacheSize=1024m"
 ```
 Then, we could
 ```
-./dev/make-distribution.sh --tgz --name hadoop-provided-scala2.11 -Dscala-2.11 -Phadoop-2.6,hadoop-provided,hive -DskipTests
+./dev/make-distribution.sh --tgz --name hadoop-provided-scala2.11 -Dscala-2.11 -Phadoop-2.6,hadoop-provided,hive,cook -DskipTests
 ```
```

Comment:
The Cook job client part in the read me looks a little out of date. Please help to fix as well.
```  
 -Dfile=target/cook-jobclient-0.1.2-snapshot.jar \
```
should be 
```
 -Dfile=target/cook-jobclient-0.1.2-SNAPSHOT.jar \
```